NopApp.Core/NopApp.Core/appsettings.json

```diff
@@ -2,6 +2,15 @@
   "JwtOptions": {
     "Secret": "Once upon a midnight dreary, while I pondered, weak and weary, over many a quaint and curious volume of forgotten lore"
   },
+  "EmailOptions": {
+    "Host": "smtp.gmail.com",
+    "Port": 587,
+    "Username": "g6.nop.app@gmail.com",
+    "Password": "Pass1234!",
+    "From": "noreply@nop.com",
+    "EmailConfirmationUrl": "https://localhost:5001/api/Authentication/Confirmation",
+    "Timeout": 20000
+  },
```

Comment:
Are you sure port 5001 is correct? We generally run the back in iis express which runs on port 7768